.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,7 +24,9 @@ Please don't include internal or private links :)
 
 ## Changelog Description
 <!--
-A description of the context of the change for a changelog. It should have a title, link to the PR, examples (if applicable), and why the change was made.
+A description of the context of the change for a changelog. It should have a title, examples (if applicable), and why the change was made.
+
+**Please keep the changelog title format same as in example bellow (### <Title>), as this is later used to generate the changelog entry title**
```

Comment:
```suggestion
**Please keep the changelog title format same as in example below (### <Title>), as this is later used to generate the changelog entry title.**
```

Reply:
Thanks @GaryJones ! I pushed new PR with the fix - https://github.com/Automattic/vip-go-mu-plugins/pull/2140